lucene/CHANGES.txt

```diff
@@ -19,7 +19,7 @@ API Changes
 * LUCENE-8474: RAMDirectory and associated deprecated classes have been
   removed. (Dawid Weiss)
 
-* LUCENE-3041: The deprecated Weight#extractTerms() method has been 
```

Comment:
Hi could you please undo all the formaating changes in the CHANGES.txt? This makes merging of other pull requests hard.
Maybe your editor reformatted the whole file. I'd just copy the new changes entry into the file.

Reply:
Yeah apparently the file has a bazillion lines that end in spaces. I typically want this on in the editor to avoid silly extra white-space in code, I'll have to figure out how to turn it off temporarily. Might not be bad to have a commit at some point that fixes this up, but we don't have to do it here.